vtex/utils/transform.ts

```diff
@@ -543,13 +545,17 @@ export const legacyFacetToFilter = (
 
     return `${link.pathname}${link.search}`;
   };
-
   return {
     "@type": "FilterToggle",
     quantity: facets.length,
     label: name,
     key: name,
     values: facets.map((facet) => {
+      if (name == "PriceRanges") {
+        facet.Map = "priceFrom";
+        facet.Value = facet.Slug!;
+      }
```

Comment:
I would prefer if you return a new facet instead of mutating the original, how that's sound?

Reply:
Its realy better, done.